invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/components/BookCard/BookCard.js

```diff
@@ -29,29 +29,30 @@ export class BookCard extends Component {
         }
         data-test={data.metadata.pid}
       >
-          <Card.Meta className={'discrete'}>
-            {data.metadata.document_type}
-          </Card.Meta>
+        <Card.Meta className={'discrete'}>
+          {data.metadata.document_type}
+        </Card.Meta>
         <Image
           centered
           src={getCover(data.metadata.pid)}
           size={'small'}
           onError={e => (e.target.style.display = 'none')}
         />
-        <Card.Content centered={'true'}>
+        <Card.Content>
```

Comment:
why not centered anymore?

Reply:
`centered` is not a property of Card.Content and was throwing an error in the console that's why I removed it.